eng/pipelines/runtime.yml

```diff
@@ -300,6 +300,7 @@ jobs:
         scenarios:
         - normal
         - wasmtestonbrowser
+        - runoutsidewasm
```

Comment:
I don't think I fully get what exactly this should be used for, would you mind adding more details? :)